account_analytic_distribution_manual/models/account_analytic_distribution_manual.py

```diff
@@ -10,6 +11,13 @@ class AccountAnalyticDistributionManual(models.Model):
 
     name = fields.Char(required=True)
     active = fields.Boolean(default=True)
+    analytic_distribution_import = fields.Json(
+        compute="_compute_analytic_distribution_import",
+        inverse="_inverse_analytic_distribution_import",
+        readonly=False,
+        string="Analytic distribution (importable)",
+        help="Defining this field will set the analytical distribution (json).",
```

Comment:
Include here an explanation about using this field for searching the analytic accounts by name.